install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+= Direct access to storage using two methods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+In OpenShift a Persistent Volume Claim (PVC) will bind to a Persistent Volume (PV) based on matching the
+spec labels of 'accessModes' and 'capacity' values. In some cases, a user may want to be specific about
+the storage they are acessing. This example will show how to a user can prebind a PV to a specifc claim 
+[METHOD 1] as well as go directly to the storage using the volume plugin [METHOD 2] inside the pod spec.
+
+In  this example we are using NFS, but the same method can be use for any of the storage plugins
+Please see the section on link:../persistent_storage/index.html[Persistent Storage Plugins] for the supports types.
+
+
+[NOTE]
+====
+All `oc ...` commands are executed on the OpenShift master node.
+====
+
+== [METHOD 1] Create the Persistent Volume using a predefined Persistent Volume Claim reference
+=== 1. Creating the persistent volume (PV) 
+The user may define the spec to include a claim name (ClaimRef) to restrict this PV to only
+bind to a particular persistent volume claim.
+
+.Persistent Volume Object Definition Using NFS
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs-pv-prebind <1>
+spec:
+  capacity:
+    storage: 1Gi <2>
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain <3>
+  ClaimRef:
+    Namespace: default <4>
+    Name: nfs-claim-prebind <5>
```

Comment:
The case of the stanza won't work in v 3.2 (AFIK). Needs to be:
  claimRef:
    name:
    namespace: